components/UserButton.vue

```diff
@@ -1,23 +1,26 @@
 <template>
   <v-menu offset-y>
     <template #activator="{ on, attrs }">
-      <v-btn icon v-bind="attrs" v-on="on">
-        <v-avatar>
-          <v-img
-            v-if="userImage"
-            :src="userImage"
-            :alt="$auth.user.Name"
-          ></v-img>
-          <v-icon v-else dark>mdi-account</v-icon>
-        </v-avatar>
-      </v-btn>
+      <div class="d-flex" v-bind="attrs" v-on="on">
+        <div class="mr-4">
+          <v-btn icon>
+            <v-avatar>
+              <v-img
+                v-if="userImage"
+                :src="userImage"
```

Comment:
Probably better to use the `imageHelper` mixin

Reply:
The main issue (for now) is that the current imageHelper mixing handles items (and a user isn't an item :/ )

I want to refactor it to clean it up and make it more modular, eventually, but I think that's outside the scope of this PR.

Reply:
Ok, sounds good.